app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -306,7 +306,8 @@ logger "Starting anaconda <%= @host %> postinstall"
 <%= snippet 'epel' -%>
 <% end -%>
 
-<%= snippet 'redhat_register' if rhel_compatible && os_major < 9 -%>
+<%= snippet 'redhat_register' if rhel_compatible && @host.operatingsystem.name != 'RedHat' -%>
+<%= snippet 'redhat_register' if @host.operatingsystem.name == 'RedHat' && os_major < 9 -%>
```

Comment:
```suggestion
<%= snippet 'redhat_register' unless @host.operatingsystem.name == 'RedHat' && os_major >= 9 -%>
```

Reply:
Wouldn't this snippet also pickup Fedora?

Reply:
Yes, but isn't that supposed to be that way? Fedora can be registered but can't use the rhsm macro.

Reply:
I'd be fine with that, I was just making sure folks were aware as the existing snippets appear to explicitly exclude `Fedora` via the `rhel_compatible` variable.